### PR TITLE
Release @google-cloud/security-center v0.3.0

### PR DESCRIPTION
**Pull request by JustinBeckwith**
This pull request was generated using releasetool.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,18 @@
 
 [1]: https://www.npmjs.com/package/@google-cloud/security-center?activeTab=versions
 
+## v0.3.0
+
+03-14-2019 07:55 PDT
+
+***This release has breaking changes!**.  The `having` enum value from `GroupFindingsRequest` has been removed.  This field was not supposed to be exposed.
```
Comment (stephenplusplus) [inline CHANGELOG.md]: I think there were some formatting errors here.

```diff
- ***This
+ **This

- breaking changes!**.
+ breaking changes!**
```

Comment (JustinBeckwith) [reply CHANGELOG.md]: Woah, NICE CATCH!

Comment (stephenplusplus) [reply CHANGELOG.md]: Woo! One last thing is that period that comes after the **. I would add a commit to change it, but I don't know if dealing with the CLA bot is more work for you than if you just edit the file yourself!

Comment (JustinBeckwith) [reply CHANGELOG.md]: don't know where my head was with this one....

Comment (bcoe) [reply CHANGELOG.md]: aha! more arguments for automation :stuck_out_tongue:

**Pull request merged**